src/report.test.js

```diff
@@ -73,7 +75,7 @@ describe('Report creation', () => {
   });
 
   test('keeps custom metrics references', () => {
-    let myMetrics = [];
+    const myMetrics = [];
```

Comment:
even though this technically works in JS, because myMetrics mutates (the later `.push`), it should be a mutable var (`let`) to flag that

Reply:
The variable itself doesn't get overwritten/mutated - a method on the object is called (`push`), so `const` is actually what we want here (and what our linters say we should do)

Reply:
Cool, I stand corrected.

Reply:
@pselle Flag it to the developer reading the code? Because it has no effect in terms of mutability in node.